docs/operators/key_value_parser.md

```diff
@@ -0,0 +1,178 @@
+## `key_value_parser` operator
+
+The `key_value_parser` operator parses the string-type field selected by `parse_from` into key value pairs. All values are of type string.
+
+### Configuration Fields
+
+| Field            | Default             | Description                                                                                                                                                                                                                              |
+| ---              | ---                 | ---                                                                                                                                                                                                                                      |
+| `id`             | `key_value_parser`  | A unique identifier for the operator                                                                                                                                                                                                     |
+| `delimiter`      | `=`                 | The delimiter used for splitting a value into a key value pair                                                                                                                                                                           |
+| `pair_delimiter` |                     | The delimiter used for seperating key value pairs, defaults to whitespace                                                                                                                                                                |
+| `output`         | Next in pipeline    | The connected operator(s) that will receive all outbound entries                                                                                                                                                                         |
+| `parse_from`     | $                   | A [field](/docs/types/field.md) that indicates the field to be parsed into key value pairs                                                                                                                                               |
+| `parse_to`       | $                   | A [field](/docs/types/field.md) that indicates the field to be parsed as into key value pairs                                                                                                                                            |
```

Comment:
The library technically still interprets `$` as `$body`, but we've gotten away from documenting it this way. Better to be clear.